podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
+# platform :ios, '8.0'
```

Comment:
@kiryteo you need to uncomment this line to make it work as well.

Reply:
Sorry, my bad.. that was the basic requirement and I left it commented. I'll change it asap.